src/client/scripts/views/CreateAccount/CreateAccountContainer.js

```diff
@@ -61,28 +61,31 @@ export class CreateAccountContainer {
         if (view === 'sign-in') {         // SignInView
             this.#viewContainer.appendChild(this.#signInViewElm);
             // URL TODO: url should be 'index.html/sign-in'
+            history.replaceState(null, "sign-in", "/index.html/sign-in");
```

Comment:
Much better, thank you. I'm getting an issue where, when I try navigating between Discover, Matches, and Settings, "index.html" keeps getting appended. I think you can look into replaceState vs pushState or something for that.
![image](https://github.com/ashleybhandari/team-friendship/assets/99090595/76c2dd3b-b960-4a34-88d2-599d5404b4e5)

Reply:
I forgot to fix the SignedIn files - I just changed it. Lmk if there's an error